tests/library/test_recipe.py

```diff
@@ -52,6 +52,46 @@ def test_standard_recipe(self):
             )
             break
 
+    def test_standard_recipe_with_additional_field(self):
+        recipe = StandardRecipe(
+            card="cards.wnli",
+            instruction=TextualInstruction(text="classify"),
+            template=InputOutputTemplate(
+                input_format="{premise}",
+                output_format="{label}",
+            ),
+            format=SystemFormat(
+                demo_format="User:{source}\nAgent:{target}\n\n",
+                model_input_format="{instruction}\n\n{demos}User:{source}\nAgent:",
+            ),
+            additional_field="test",
```

Comment:
Whta if people will start adding fields that will not be compatible with different changes.
Perhaps, we will require a prefix like 'custom_field_override_serializer".